Donation-platform/src/main/java/com/google/model/Charity.java

```diff
@@ -86,4 +91,22 @@ public Double getTrendingScoreCharity() {
   public void setTrendingScoreCharity(Double trendingScore) {
     this.trendingScore = trendingScore;
   }
+
+  public Double getUserRating() {
+    return userRating;
+  }
+
+  public void setUserRating( Double userRating){
+    this.userRating = userRating;
+  }
+}
+
+class SortByTrending implements Comparator<Charity> {
```

Comment:
Replace this SortByTrending class with the following method:
`   @Override
    public int compareTo(Charity b) {
        return Double.compare(b.getTrendingScoreCharity(), this.getTrendingScoreCharity());
    }`

Reply:
^ Change (2) for using Comparable